tests/RobotFramework/plugin_apt/improve_tedge_apt_plugin_error_messages.robot

```diff
@@ -0,0 +1,65 @@
+#Command to execute:    robot -d \results --timestampoutputs --log improve_tedge_apt_plugin_error_messages.html --report NONE --variable VERSION:0.7.4 --variable HOST:192.168.1.130 /thin-edge.io-fork/tests/RobotFramework/plugin_apt/improve_tedge_apt_plugin_error_messages.robot
+*** Settings ***
+Library    Browser
+Library    OperatingSystem
+Library    Dialogs
+Library    SSHLibrary
+Library    DateTime
+Library    CryptoLibrary    variable_decryption=True
+Suite Setup            Open Connection And Log In
+Suite Teardown         SSHLibrary.Close All Connections
+
+*** Variables ***
+${HOST}           
+${USERNAME}       pi
+${PASSWORD}       crypt:LO3wCxZPltyviM8gEyBkRylToqtWm+hvq9mMVEPxtn0BXB65v/5wxUu7EqicpOgGhgNZVgFjY0o=  
+${VERSION}        0.7.4
```

Comment:
I feel it is problematic in the future. When we release new version, we always need to update this hard-coded version?